docs/conf.py

```diff
@@ -81,6 +83,11 @@
 .. |components_type| replace:: Union[:class:`disnake.ui.ActionRow`, :class:`disnake.ui.WrappedComponent`, List[Union[:class:`disnake.ui.ActionRow`, :class:`disnake.ui.WrappedComponent`, List[:class:`disnake.ui.WrappedComponent`]]]]
 .. |resource_type| replace:: Union[:class:`bytes`, :class:`.Asset`, :class:`.Emoji`, :class:`.PartialEmoji`, :class:`.StickerItem`, :class:`.Sticker`]
 .. _coroutine_link: https://docs.python.org/3/library/asyncio-task.html#coroutine
+.. |data_class| replace:: This class is a :ref:`Data Class <data_class>`
+.. |data_class_constructible| replace:: This class is a constructible :ref:`Data Class <data_class>`
+.. |discord_abc| replace:: This class is an :ref:`ABC <disnake_abc>`
+.. |discord_model| replace:: This class is a :ref:`Discord Model <discord_model>`
+.. |discord_enum| replace:: This class is an :ref:`Enumeration <discord_enum>`
```

Comment:
They are needed because previously, i.e. before the split up thing, all classes, data classes, events etc. were on the same page, under the same heading. For example, All of the Discord Models were under their respective section. Now, there is no just one section for all e.g. models, but rather a few dozens of them, a
but we still want users to know what is a model, "what is what"

Reply:
Ah, I see. In that case, could we make it easier to tell them apart from the actual description (maybe just italic text) and add periods at the end?

Reply:
Periods work fine, but adding italics somehow breaks `:ref:`s. I'll try to fix that